test/labeling/model/test_baseline.py

```diff
@@ -11,7 +12,7 @@
 
 class BaselineModelTest(unittest.TestCase):
     def test_random_vote(self):
-        L = np.array([[1, 2, 1], [0, 4, 3], [3, 0, 0], [1, 2, 2]])
+        L = csr_matrix(np.array([[1, 2, 1], [0, 4, 3], [3, 0, 0], [1, 2, 2]]))
```

Comment:
Instead of having np.array as an intermediate step, why not construct the csr_matrix directly from the list of lists? (same comment for other two occurrences in this file).